src/app/(proper_react)/redesign/(authenticated)/user/dashboard/Dashboard.stories.tsx

```diff
@@ -22,7 +22,8 @@ const brokerOptions = {
   "emtpy-scan-in-progress": "Scan is in progress with no results",
   unresolved: "With unresolved scan results",
   "unresolved-scan-in-progress": "Scan is in progress with unresolved results",
-  resolved: "All scan results resolved",
+  resolved: "Some scan results resolved",
+  allResolved: "All scan results resolved",
```

Comment:
That's not my understanding. I think those are counted as "in progress"?

Reply:
Ah I suppose, they just show up on the "Fixed" tab, since we don't have an "In progress" tab. Could you then maybe rename `resolved` to `inProgress`? Then we're testing three cases: with unresolved scan results, with in-progress scan results, and all scan results done. Which I guess might also warrant another whole slew of Storybook stories, but let's not.

Reply:
I thought about creating the stories for the "inProgress" cases.. and decided to ignore it at this time 😅

Reply:
Renamed.